RescheduleAppointments/force-app/main/default/lwc/mobileAppointmentBookingResourceAssignment/mobileAppointmentBookingResourceAssignment.html

```diff
@@ -1,22 +1,23 @@
 <template>
-    <div class="container">
-        <div class="title">{resourceAssignmentTitle}</div>
-        <div class="separator"></div>
-        <template if:true={showExcludedMsg}>
-            <div class="mobile-worker-content">{excludedMsg}</div>
-        </template>
-        <template if:false={showExcludedMsg}>
-            <div>
-                <lightning-radio-group name="resourceAssignment"
-                          aria-label={resourceAssignmentTitle}
-                          options={options}
-                          value={value}
-                          type="radio"
-                          onchange={handleAssignmentMethodChange}
-                          >
-                </lightning-radio-group>
-            </div>
-        </template>
-    </div>
-    
+    <template lwc:if={showMobileWorkerChoice}>
+        <div class="container">
+                <div class="title">{resourceAssignmentTitle}</div>
+                <div class="separator"></div>
+                <template lwc:if={showExcludedMsg}>
+                    <div class="mobile-worker-content">{excludedMsg}</div>
+                </template>
+                <template lwc:else>
+                    <div>
+                        <lightning-radio-group name="resourceAssignment"
+                                aria-label={resourceAssignmentTitle}
+                                options={options}
+                                value={value}
+                                type="radio"
+                                onchange={handleAssignmentMethodChange}
+                                >
```

Comment:
`>` should be at the end of the previous line: `onchange={handleAssignmentMethodChange}>`